tests/src/testklshell.cpp

```diff
@@ -74,7 +74,7 @@ static auto NonLinearKLShellLoadControlTR() {
     return fext;
   };
 
-  using ElementType = Ikarus::AutoDiffFE<Ikarus::KirchhoffLoveShell<decltype(basis)>>;
+  using ElementType = KirchhoffLoveShell<decltype(basis)>;
```

Comment:
Thanks! Can you also change this in `testadaptivestepsizing.cpp` ? Does it reduce the time significantly such that it can be moved to `cpp_quick` as well?